dlrs/main/pages/managelookuprollupsummaries_New.page

```diff
@@ -0,0 +1,264 @@
+<apex:page controller="ManageLookupRollupSummariesNewController" lightningStylesheets="true" tabStyle="ManageLookupRollupSummaries__tab" showHeader="true" sidebar="true" action="{!init}">
```

Comment:
Will we continue to have two setup pages? 
If yes, then please remove the `_` or rename to: ManageLookupRollupSummariesNew

reason: no underscores in metadata names